verifier/src/mbedtls.rs

```diff
@@ -322,7 +322,7 @@ mod test {
             .collect::<Vec<_>>();
         let verifier = MbedTlsCertificateChainVerifier::new(trust_anchor);
         assert!(verifier
-            .verify_certificate_chain(chain.iter(), crls.iter(), DateTime::INFINITY)
+            .verify_certificate_chain(chain.iter(), crls.iter(), None)
```

Comment:
It looks to me like we're skipping time validation here anyway, so this should be fine

Reply:
that's correct, Mbedtls will only verify the time if MBEDTLS_HAVE_DATE_TIME is set, https://github.com/Mbed-TLS/mbedtls/blob/5edb9427086a4cdc8e567bb8069db9d09a58e21b/library/x509.c#L1109

Our build leaves it undefined, https://github.com/mobilecoinfoundation/rust-mbedtls/blob/mc-develop/mbedtls-sys/build/config.rs#L74

Mbedtls also uses a function pointer to get the time value when it verifies the certificates